src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -200,18 +323,21 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             var context = await GetOrCreateContext(document, position, cancellationToken).ConfigureAwait(false);
             options = GetUpdatedRecommendationOptions(options, document.Project.Language);
 
+            var typeInferenceService = document.GetLanguageService<ITypeInferenceService>();
+            var inferredTypes = typeInferenceService.InferTypes(context.SemanticModel, position, cancellationToken);
```

Comment:
@CyrusNajmabadi Do you think I need to do more than the position-based thing here for a first go? If so, how do I generally find/produce an expression of the right kind for the location? Is there a helper for that?